### PR TITLE
Add mock message of the day

### LINKED ISSUE #884: Display message of the day
The users are looking forward to the timeline displaying a message of the day.

There were thoughts that this could show issues to focus on, but the client lead was keen to see `Manager of the day: Daisy`.

We still need to work on layout/design for the timeline page, but a large central text box occupying 50% of width would be a start.

The table will be implemented in this issue: https://github.com/debrief/pepys-import/issues/871

It is expected that a call to the options be made just after the existing update cycle.

### PR DESCRIPTION
**Pull request by krsnik93**
## 🧰 Issue

Fixes #884 

## 🚀 Overview: 

Introduces a horizontally centered heading between the toolbar and the timelines. Heading will display whatever is returned in `config.MessageOfTheDay` or a, just for now, hardcoded `Manager of the day: Daisy Duke`.

## 🤔 Reason: 

Enables personalisation and provides more info to the user.
 
## 🔨Work carried out:

- [x] Tests pass

## 🖥️ Screenshot

image

## Confirmations

- [x] I have chosen reviewers for my PR.
- [ x I have chosen an appropriate label for the PR, adding `interactive_review` if reviewers will need to see UI
- [ ] I have extended/updated the documentation in `\docs` folder
- [ ] Any database content changes (Create, Edit, Delete) are recorded in the Log/Changes tables
- [ ] Any database schema changes are implemented via `alembic revision` [transitions](https://pepys-import.readthedocs.io/en/latest/database_migration.html#how-to-use-it-for-developers)
- [x] I have completed the mandatory sections of this document.
- [x] I have deleted any unused sections.

### CODE REVIEW

**Comment by IanMayo** [comment]
Thanks @krsnik93 

* [ ] I'm worried about the impact on vertical space.  We've been told that there could be up to 25 participants in a serial.  So, could the MotD and the time period be moved up into the top bar please? (layout thoughts below)
* [ ] The backend data should be there for you to connect to, as covered in [this](https://github.com/debrief/pepys-import/issues/871) issue. Since you want the interval and the MotD I _guess_ you could `select *` from the options table
* [ ] We're lacking a bit of contrast in the MotD. Could it be in a block with a white background please?
* [ ] (very minor) I've removed Daisy's surname in the original issue - to be more representative of the space required. Could you remove the surname in here too please?

## Layout thoughts
We've lost our UX designer, though she did leave us with a layout for the timeline:
image

We're still missing some elements, but that's the general direction I'd like to take.  So, a single _row_ of elements along the top, with the rest left for the serials.

**Review by IanMayo — COMMENTED** [review]
Changes requested [here](https://github.com/debrief/pepys-import/pull/904#issuecomment-835167886)

**Comment by krsnik93** [comment]
@IanMayo Are you ok with the toolbar (including date pickers, countdown timer, and whatever comes next) staying? It does not take up too much in terms of height.

I could introduce the first row from above picture, adding date and time display and message of the day, while leaving mocks for the 'Data coverage per day' and 'Data processed' for now...

**Comment by IanMayo** [comment]
Hello @krsnik93 - I'd be tempted to combine the title bar with the control panel.

We have ASCII art for the portrait of Samuel Pepys that we can use as "Title" at top-left. I can provide it as a new issue.  

Let's not have the mockups. They were useful when we had no useful information - but now that we're delivering _real_ data, let's stick to real data on the dashboard.

**Comment by krsnik93** [comment]
@IanMayo new look:

image
image

Widths will look a bit nicer when we add the inner two boxes from the design layout as well.

File: pepys_timeline/static/js/src/pepys.js [hunk 1]
```diff
@@ -1,10 +1,15 @@
+/* eslint-env browser */
+
 moment.locale("en");
 
 const DATE_FORMATS = {
   visavail: "YYYY-MM-DD HH:mm:ss",
   metadata: "YYYY-MM-DD",
   picker: "DD/MM/YYYY",
 }
+const DEFAULT_MESSAGE_OF_THE_DAY = 'Manager of the day: Daisy Duke';
```
Comment (IanMayo) [inline pepys_timeline/static/js/src/pepys.js:10]: If we don't have a database connection, or can't get the field from the database, then let's have the default message say something valid `Message of the day: [PENDING]`

Comment (krsnik93) [reply pepys_timeline/static/js/src/pepys.js]: Noted!

**Review by IanMayo — APPROVED** [review]

**Pull request merged**